src/vaadin-select.html

```diff
@@ -567,7 +567,7 @@
           const hasContent = this._hasContent(selected);
 
           // Toggle visibility of _valueElement vs fallback input with placeholder
-          this._valueElement.slot = hasContent ? 'value' : '';
+          this._valueElement.slot = hasContent ? 'input' : '';
```

Comment:
same as above, should be conditionally set based on the text-field version used

Reply:
Updated. Detecting slot existence now.